engine/interpreter-dsl-test/src/test/java/org/enso/interpreter/dsl/test/ThrowableCatchTest.java

```diff
@@ -86,37 +72,6 @@ public void tearDown() {
     ctx.close();
   }
 
-  @Test
-  public void testMostRuntimeExceptionsCanPropagateFromBuiltinMethods() {
```

Comment:
How this is related to `ydoc-server`?

Reply:
See https://github.com/enso-org/enso/pull/10156#discussion_r1630975940. `interpreter-dsl-test` got broken because of the changes with modules. Moreover, `interpreter-dsl-test` project was never setup to properly work with modules, the tests there were invoked with plain old class-path.